backend/utils/memories/process_memory.py

```diff
@@ -52,15 +52,16 @@ def _get_structured(
             return summarize_open_glass(memory.photos), False
 
         # from Friend
+        tz = notification_db.get_user_time_zone(uid)
         if force_process:
             # reprocess endpoint
-            return get_transcript_structure(memory.get_transcript(False), memory.started_at, language_code), False
+            return get_transcript_structure(memory.get_transcript(False), memory.started_at, language_code,tz), False
```

Comment:
pls check all usages of func `get_transcript_structure` in the backend side, ensure all of them uptodate with new params: tz of tz should be an optional param?

Reply:
There are multiple usages of `get_transcript_structure` (which mostly are old funcs), confirmed with Joan that only v2/memories endpoint is being used for memories. Anyway will update all of them just to be on the safe side.

tz is required for events to be in correct timezone, and we can never be sure if a conversation will contain an event or not before calling `get_transcript_structure`, so made it mandatory